lib/deploy/deploy-support.js

```diff
@@ -291,15 +291,26 @@ async function makeService({
       service = await fc.getService(serviceName);
     } catch (ex) {
       
-      if (ex.code === 'AccessDenied' || ex.code === 'ERR_INVALID_CHAR' || ex.code === 'ENOTFOUND') {
-        console.error(red(`The accountId entered is incorrect. We speculate that you used the subuser uid, login name, login name prefix, display name or thers\n`+
-        `please enter the correct accountId, which is the account ID under your primary account. You can reset your account ID by executing fun config\n`+
-        `For the method of taking the primary account ID, please refer to https://account.console.aliyun.com/#/secure`));
+      if (ex.code === 'AccessDenied') {
+
+        if (ex.message.indexOf('the caller is not authorized to perform') !== -1) {
```

Comment:
The caller is not authorized to perform this action. 

Maybe you can grant AliyunRAMFullAccess policy to the sub-account or use the primary account. You can refer to Chinese doc https://github.com/aliyun/fun/blob/master/docs/usage/faq-zh.md#nopermissionerror-you-are-not-authorized-to-do-this-action-resource-acsramxxxxxxxxxxrole-action-ramgetrole or English doc https://github.com/aliyun/fun/blob/master/docs/usage/faq.md#nopermissionerror-you-are-not-authorized-to-do-this-action-resource-acsramxxxxxxxxxxrole-action-ramgetrole for help.

If you don’t want use the AliyunRAMFullAccess policy or primary account, you can also specify the Role property for Service. You can refer to Chinease doc https://github.com/aliyun/fun/blob/master/docs/specs/2018-04-03-zh-cn.md#aliyunserverlessservice or Enaglish doc https://github.com/aliyun/fun/blob/master/docs/specs/2018-04-03.md#aliyunserverlessservice for help.